<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1051</width>
      <height>831</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Criar Questão</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_questionText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>1011</width>
        <height>141</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Número real&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Use ponto e não vírgula&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Exemplos corretos:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;1/137&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;3.14159&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;6.02E-23&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;9E9&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="placeholderText">
       <string>Main question text goes here</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_variables">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>41</y>
        <width>641</width>
        <height>151</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Arraste com o mouse para mudar a ordem dos itens.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Clique duplo para apagar um item.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_varName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Nome da Variável &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Apenas caracteres alfanuméricos e sem espaço. Não iniciar com números.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="placeholderText">
       <string>Var Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_varMin">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Valor mínimo a ser gerado&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;User ponto e não vírgula&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Notação científica com E no lugar de 10^.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Exemplos:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;3.14159&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;1/137&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;6.02E23&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;3E9&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="placeholderText">
       <string>Var Min Value</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_varMax">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Var Max Value</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_varStep">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Var Step</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_preview">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>740</y>
        <width>1011</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Preview</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/preview.png</normaloff>icons/preview.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_images">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>10</y>
        <width>191</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>1031</width>
        <height>371</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Questão Aberta</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_addSolution">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>10</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Solution</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/plus_3.png</normaloff>icons/plus_3.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_itemText">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>841</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Item Text</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_itemSolVar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Var Solution Item </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_itemSolTex">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>40</y>
          <width>601</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Item Text</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget_solutions">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>1011</width>
          <height>261</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Arraste com o mouse para mudar a ordem dos itens.
Clique duplo para apagar um item.</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Múltipla Escolha</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Verdadeiro ou Falso</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>40</y>
        <width>131</width>
        <height>151</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="pushButton_addVar">
       <property name="text">
        <string>Var</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/plus_3.png</normaloff>icons/plus_3.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_addImage">
       <property name="text">
        <string>img</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/image_3.png</normaloff>icons/image_3.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_checkVars">
       <property name="text">
        <string>Vars</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/check.png</normaloff>icons/check.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox_Rline">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>10</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>sample</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>function</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_function">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Equation or Function with R sintax</string>
      </property>
     </widget>
     <zorder>lineEdit_function</zorder>
     <zorder>textEdit_questionText</zorder>
     <zorder>listWidget_variables</zorder>
     <zorder>lineEdit_varName</zorder>
     <zorder>lineEdit_varMin</zorder>
     <zorder>lineEdit_varMax</zorder>
     <zorder>lineEdit_varStep</zorder>
     <zorder>pushButton_preview</zorder>
     <zorder>listWidget_images</zorder>
     <zorder>tabWidget_2</zorder>
     <zorder>splitter</zorder>
     <zorder>comboBox_Rline</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Criar Atividade</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_atividade">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>281</width>
        <height>201</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Arraste com o mouse para mudar a ordem dos itens.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Clique duplo para apagar um item.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWebEngineView" name="webEngineView_previewAtividade">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>731</width>
        <height>771</height>
       </rect>
      </property>
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_busca">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Digite para buscar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addQuestion">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/plus_3.png</normaloff>icons/plus_3.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Disciplina: Todas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fisica I</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Física II</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Física III</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mecanica</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ondulatória</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Dificuldade: Todos</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>*</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>**</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>***</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Tópicos: Todos</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B</string>
       </property>
      </item>
     </widget>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>281</width>
        <height>321</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>XML</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_moodle">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>251</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/gerar.png</normaloff>icons/gerar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nomeAtividade">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>251</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Nome da Atividade</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nomeDisciplina">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>251</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Disciplina</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Quantidade</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_nAtividades">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nomeArquivo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>251</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>File Name</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>PDF</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_previewQuestion">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/preview.png</normaloff>icons/preview.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Tipos: Todos</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Teórica</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Conceitual</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Numérica</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
    <addaction name="actionRexams"/>
    <addaction name="actionLatex"/>
    <addaction name="actionMath_Equations"/>
    <addaction name="actionR_functions"/>
    <addaction name="actionEste_Software"/>
   </widget>
   <widget class="QMenu" name="menuConfiguracoes">
    <property name="title">
     <string>Configuracoes</string>
    </property>
    <widget class="QMenu" name="menuPasta_de_Quest_es">
     <property name="title">
      <string>Pasta de Questões</string>
     </property>
     <addaction name="actionRaiz"/>
    </widget>
    <addaction name="actionSemente"/>
    <addaction name="menuPasta_de_Quest_es"/>
    <addaction name="actionLingua"/>
   </widget>
   <addaction name="menuConfiguracoes"/>
   <addaction name="menuSobre"/>
  </widget>
  <action name="actionSemente">
   <property name="text">
    <string>Semente</string>
   </property>
  </action>
  <action name="actionRexams">
   <property name="text">
    <string>Rexams</string>
   </property>
  </action>
  <action name="actionLatex">
   <property name="text">
    <string>Latex</string>
   </property>
  </action>
  <action name="actionMath_Equations">
   <property name="text">
    <string>Math Equations</string>
   </property>
  </action>
  <action name="actionR_functions">
   <property name="text">
    <string>R functions</string>
   </property>
  </action>
  <action name="actionEste_Software">
   <property name="text">
    <string>Este Software</string>
   </property>
  </action>
  <action name="actionRaiz">
   <property name="text">
    <string>Raiz</string>
   </property>
  </action>
  <action name="actionLingua">
   <property name="text">
    <string>Lingua</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
